client/client_test.go

```diff
@@ -14,18 +15,25 @@ func TestSignAndBroadcastTx(t *testing.T) {
 	signer, err := GenerateSigner()
 	require.NoError(t, err)
 
-	client, err := NewClient("http://localhost:26657")
+	client, err := NewClient("http://localdev.me:26657")
```

Comment:
Why the change? Don't these resolve to the same thing?

Reply:
Yup they totally do. Reverting.